docs/commands.md

```diff
@@ -0,0 +1,86 @@
+<!--[metadata]>
++++
+title = "Understand the typical commands"
+description = "Description of key notary commands"
+keywords = ["docker, notary, notary-client"]
++++
+<![end-metadata]-->
+
+# Understand the typical notary commands
+
+## Initialize a collection
+`notary init example.com/image`
```

Comment:
That works. We should also define what a "GUN" is at the top. It's also defined elsewhere but a cheatsheet should be as self contained as possible.